js/editor-ta.js

```diff
@@ -8,14 +8,20 @@ function EditorTextArea(editorElement, settings) {
   this.settings_ = settings;
   this.numLines_ = 1;
   this.verticalTextareaPadding_ = 20;
-  this.horiztonalTextareaPadding_ = 8;
+  this.horizontalTextareaPadding_ = 8;
   this.currentSession_ = null;
   this.dimentions = {
```

Comment:
how is dimensions.height different from calculatedHeight? maybe comment on the var declaration